packages/wrangler/src/dev.tsx

```diff
@@ -726,43 +731,43 @@ type useHotkeysInitialState = {
 function useHotkeys(initial: useHotkeysInitialState, port: number) {
   // UGH, we should put port in context instead
   const [toggles, setToggles] = useState(initial);
-  useInput(
-    async (
-      input,
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      key
-    ) => {
-      switch (input) {
-        case "b": // open browser
-          await open(`http://localhost:${port}/`);
-          break;
-        case "d": // toggle inspector
-          await open(
-            `https://built-devtools.pages.dev/js_app?experiments=true&v8only=true&ws=localhost:9229/ws`
-          );
-          break;
-        case "s": // toggle tunnel
-          setToggles((previousToggles) => ({
-            ...previousToggles,
-            tunnel: !previousToggles.tunnel,
-          }));
-          break;
-        case "l": // toggle local
-          setToggles((previousToggles) => ({
-            ...previousToggles,
-            local: !previousToggles.local,
-          }));
-          break;
-        case "q": // shut down
-        case "x": // shut down
-          process.exit(0);
-          break;
-        default:
-          // nothing?
-          break;
-      }
-    }
-  );
+  // useInput(
+  //   async (
+  //     input,
+  //     // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  //     key
+  //   ) => {
+  //     switch (input) {
+  //       case "b": // open browser
+  //         await open(`http://localhost:${port}/`);
+  //         break;
+  //       case "d": // toggle inspector
+  //         await open(
+  //           `https://built-devtools.pages.dev/js_app?experiments=true&v8only=true&ws=localhost:9229/ws`
+  //         );
+  //         break;
+  //       case "s": // toggle tunnel
+  //         setToggles((previousToggles) => ({
+  //           ...previousToggles,
+  //           tunnel: !previousToggles.tunnel,
+  //         }));
+  //         break;
+  //       case "l": // toggle local
+  //         setToggles((previousToggles) => ({
+  //           ...previousToggles,
+  //           local: !previousToggles.local,
+  //         }));
+  //         break;
+  //       case "q": // shut down
+  //       case "x": // shut down
+  //         process.exit(0);
+  //         break;
+  //       default:
+  //         // nothing?
+  //         break;
+  //     }
+  //   }
+  // );
```

Comment:
Is this supposed to be commented out?

Reply:
Damn sorry that was me trying to fix something else... Will redo and repush